lib/lit/import.rb

```diff
@@ -136,21 +137,19 @@ def upsert(locale, key, value) # rubocop:disable Metrics/MethodLength
         # is the array
         val = value.is_a?(Array) ? [value] : value
         I18n.t(key, default: val)
+      end
 
-        # this indicates that this translation already exists
-        existing_translation =
-        Lit::Localization.joins(:locale, :localization_key)
-                         .find_by('localization_key = ? and locale = ?', key, locale)
-
-        return unless existing_translation
```

Comment:
existing_translation might be `nil`, as there might be key in imported files, that has been added to `Lit.ignored_keys` array (in initializer)